data/software_stack/eessi.yaml

```diff
@@ -0,0 +1,47 @@
+jupyterhub::kernel::venv::python: /cvmfs/pilot.eessi-hpc.org/latest/init/eessi_pilot_python3
```

Comment:
This is still waiting on integration into EESSI in https://github.com/EESSI/software-layer/pull/43